templates/patterns/featured_item/pattern-featured-item.html.twig

```diff
@@ -0,0 +1,15 @@
+{#
+/**
+ * @file
+ * Field.
+ */
+#}
+{% include '@ecl/ec-component-featured-item' with {
+  'variant': variant,
+  'image': image,
+  'metas': metas,
+  'title': title,
+  'link': link,
+  'extra_classes': extra_classes,
+  'extra_attributes': extra_attributes,
```

Comment:
See comment above, remove `extra_classes` and `extra_attributes`.

Reply:
Removed.